doc/INSTALL-Win64.txt

```diff
@@ -0,0 +1,178 @@
+-------------------------------------------------------------------------------
+KVIrc4 win64 compiling howto                                          by Cizzle
+ based on wodim's guide at https://thacid.wordpress.com/2010/10/18/compiling-kvirc-64-bits-from-svn-using-visual-studio-2008/
+ and DarthGandalf's appveyor buildinstructions
+-------------------------------------------------------------------------------
+
+This paper describes how to successfully compile a working copy of KVIrc 4.3.x
+under Microsoft Windows operating systems. For own compilations we assume a
+root-directory C:\kvirccomp\ being used.
+
+-------------------------------------------------------------------------------
+Prerequisites
+-------------------------------------------------------------------------------
+
+- A 64-bit Microsoft Windows operating system.
```

Comment:
I think you can cross-compile from 32 bits to 64 bits. I didn't test it for kvirc though, and the only possible problem I foresee, is perl support, because perl is used for both docs and scripting, so it needs to run and to link.